e2e-tests/support/tasks.js

```diff
@@ -1,13 +1,24 @@
 const crypto = require('node:crypto');
 const db = require('./db-client');
+const redis = require('./redis-client');
 
 module.exports = {
-  createTasks: ({ dbName, dbConnectionString }) => {
+  createTasks: ({
+    dbName,
+    dbConnectionString,
+    redisHost,
+    redisPort,
+    redisKey,
+  }) => {
```

Comment:
Maybe could restructure this file - similar comment as above and then would be easier to add documentation